js/src/offcanvas.js

```diff
@@ -266,7 +268,10 @@ EventHandler.on(window, EVENT_LOAD_DATA_API, () => {
 EventHandler.on(window, EVENT_RESIZE, () => {
   for (const element of SelectorEngine.find('[aria-modal][class*=show][class*=offcanvas-]')) {
     if (getComputedStyle(element).position !== 'fixed') {
-      Offcanvas.getOrCreateInstance(element).hide()
+      const data = Offcanvas.getOrCreateInstance(element)
+      if (data._config.hideNotFixedOnWindowResizing) {
```

Comment:
Why are you using the `_config` here? Isn't there another property that doesn't start with an underscore to use?

In the future we might mangle all properties that start with an underscore so this is a bad practice (let alone that it's a common convention for private stuff).

Reply:
Hmm, indeed it might be OK because we inherit from the base component.

We might want to revisit this later @GeoSot :)

Reply:
As you already wrote, and I am pretty sure, I have commented in another similar approach, we should avoid referring to class 'private' properties.
It of course work on this code, but even if we mangle the properties or use native [private members](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields), we will not be able to replicate this behavior

Reply:
The thing is that we currently use this everywhere and this PR just follows the same scheme.

How else can this be done? And we should probably try to do the same across the whole codebase in a separate PR